string_assignment.js

```diff
@@ -0,0 +1,153 @@
+// # String Exercises
+//
+// 1. `console.log` the first character of a string.
+let leaguer = "boyd";
+console.log(leaguer[0]);
+// 2. `console.log` the length of a string.
+console.log(leaguer.length);
+// 3. `console.log` the last character of any string.
+console.log(leaguer[leaguer.length-1]);
+// 4. Create a code block that takes a single string variable, and returns a copy of the string with the last letter capitalized. For example:
+// 'jimmy'
+// // => 'jimmY'
+console.log(leaguer[0] + leaguer[1] + leaguer[2] + leaguer[leaguer.length-1].toUpperCase());
```

Comment:
This would be instance of hard coding where if we change leaguer to "Jackie Robinson", your code wouldn't give desired output. Try to avoid this in future assignments. Your code must work with differing input and still give us consistent and correct output.